go-controller/pkg/cni/helper_linux.go

```diff
@@ -169,7 +170,7 @@ func setupNetwork(link netlink.Link, ifInfo *PodInterfaceInfo) error {
 		}
 	}
 	for _, route := range ifInfo.Routes {
-		if err := cniPluginLibOps.AddRoute(route.Dest, route.NextHop, link, ifInfo.RoutableMTU); err != nil {
+		if err := cniPluginLibOps.AddRoute(route.Dest, route.NextHop, link, ifInfo.RoutableMTU); err != nil && !os.IsExist(err) {
```

Comment:
will remove this. This is only needed for downstream: because for some specific purpose we'd need to annotate the route for layer2/localnet network, which would result in this particular error because it conflict with source based routes of those directly connected link.